### PR TITLE
Titan AT

### PR DESCRIPTION
**Pull request by synixebrett**
Requires #45

### CODE REVIEW

**Comment by nameless** [comment]
Should probably just include various other locking types, NLAW, Javelin etc.

**Comment by synixebrett** [comment]
I'm fine with not, mainly because the Javelin is basically the same, and no one uses the PLOS functionality of the NLAW.

File: guides/players/titan-at.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Anti-Tank (AT) missiles require a thermal target to lock on to, and will seek the target after launch.
+
+1. Enter thermal view with `N`
+2. Center target in view finder
+3. Hold `Tab`
+4. Wait for lock tone and `SEEK`
+5. Fire with `Left Click`
+
+![Diagram](img/titan_locking.png)
```
Comment (nameless) [inline guides/players/titan-at.md]:
```suggestion
![Diagram](img/titan_locking.png)  
```
Line break

File: guides/players/titan-at.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Missile Locking
+
+![Diagram](img/titan_locked.png)
```
Comment (nameless) [inline guides/players/titan-at.md]:
```suggestion
![Diagram](img/titan_locked.png)  
```
Line break

File: guides/players/titan-at.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+Missile Locked
+
+### Anti-Personnel
+
+Anti-Personnel (AP) missiles are fly-by-wire. They are fired without a locked target and continue towards the center of the view finder.
+
+1. Center target in view finder
```
Comment (nameless) [inline guides/players/titan-at.md]:
```suggestion
1. Center target in viewfinder
```
[Source](https://www.merriam-webster.com/dictionary/viewfinder)

File: guides/players/titan-at.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+2. Fire with `Left Click`
+3. Track target with view finder
```
Comment (nameless) [inline guides/players/titan-at.md]:
```suggestion
3. Track target with viewfinder
```

**Pull request merged**